mods/ra/rules/aircraft.yaml

```diff
@@ -118,6 +118,7 @@ MIG:
 		Speed: 223
 		RepulsionSpeed: 40
 		MaximumPitch: 56
+		LandableTerrainTypes: Clear,Rough,Road,Beach
```

Comment:
All aircraft in RA except transport helis have `LandWhenIdle: false` and can only land on airfields/helipads or repair depots, and the terrain type shouldn't matter there, so these yaml changes should be unnecessary.

Reply:
Ok. I've added an extra check in `CanLand` so this hack is not necessary anymore. Removed `LandableTerrainTypes` from non-vtol craft. I've left it in for helicopters. I guess that's just a matter of taste. I will remove it if you find it really objectionable.

Reply:
See my comment on cnc above.